### PR TITLE
Refactor a bit to simplify Embeddings

### PR DESCRIPTION
**Pull request by pradh**
Instead of having the interface sort variables, it just returns the sorted sentence entries.

This will help simplify the merging across indexes (for custom DC), and reduce logic duplication in EmbeddingsStore implementations.

Note: This PR has no functionality change.

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
nice!

File: nl_server/search.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+_NUM_SV_INDEX_MATCHES = 40
+# Number of matches to find within the SV index if skipping topics.
+# Reason: Since we're going to drop a few candidates at the top,
+# try to retrieve more from vector DB.
+_NUM_SV_INDEX_MATCHES_WITHOUT_TOPICS = 60
+
+
+#
+# Given a list of query embeddings, searches the embeddings index
+# and returns a list of candidates in the same order as original queries.
```
[the remark this answers is not in the document]
Comment (pradh) [reply nl_server/search.py]: Done

File: nl_server/search.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Google LLC
```
[the remark this answers is not in the document]
Comment (pradh) [reply nl_server/search.py]: Done

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Pull request merged**